SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -135,39 +118,40 @@ FSpatialNetGUIDCache::FSpatialNetGUIDCache(USpatialNetDriver* InDriver)
 {
 }
 
-FNetworkGUID FSpatialNetGUIDCache::AssignNewEntityActorNetGUID(AActor* Actor, const ::worker::Map< std::string, std::uint32_t >& SubobjectToOffset)
+FNetworkGUID FSpatialNetGUIDCache::AssignNewEntityActorNetGUID(AActor* Actor, const SubobjectToOffsetMap& SubobjectToOffset)
 {
-	FEntityId EntityId = Cast<USpatialNetDriver>(Driver)->GetEntityRegistry()->GetEntityIdFromActor(Actor);
-	check(EntityId.ToSpatialEntityId() > 0);
+	USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(Driver);
+
+	Worker_EntityId EntityId = SpatialNetDriver->GetEntityRegistry()->GetEntityIdFromActor(Actor);
+	check(EntityId > 0);
```

Comment:
Worker_EntityId is an int, are negative values really invalid? Or just != 0?

Reply:
They can't be negative, probably can be changed to be `== INVALID`